client/web/src/search/home/SearchPage.tsx

```diff
@@ -77,29 +75,19 @@ export const SearchPage: React.FunctionComponent<React.PropsWithChildren<SearchP
                     [styles.panelsContainerWithCollaborators]: showCollaborators,
                 })}
             >
-                {props.isSourcegraphDotCom && !props.authenticatedUser && <LoggedOutHomepage {...props} />}
-                {props.isSourcegraphDotCom && props.authenticatedUser && !showEnterpriseHomePanels && (
-                    <TipsAndTricks
-                        examples={exampleTripsAndTricks}
-                        moreLink={{
-                            label: 'More search features',
-                            href: 'https://docs.sourcegraph.com/code_search/explanations/features',
-                            trackEventName: 'HomepageExampleMoreSearchFeaturesClicked',
-                        }}
-                        {...props}
-                    />
-                )}
+                {props.isSourcegraphDotCom && !props.authenticatedUser && <LoggedOutHomepage {...props} queryState={queryState} setQueryState={setQueryState} />}
 
                 {showEnterpriseHomePanels && props.authenticatedUser && (
                     <HomePanels showCollaborators={showCollaborators} {...props} />
                 )}
 
-                {!showEnterpriseHomePanels && !props.isSourcegraphDotCom && (
+                {(!showEnterpriseHomePanels && !props.isSourcegraphDotCom) || (props.isSourcegraphDotCom && props.authenticatedUser && !showEnterpriseHomePanels) && (
```

Comment:
I think we can simplify this? Not identical but I don't think we have (many?) customers allowing unauthenticated use.

```suggestion
                {(!showEnterpriseHomePanels && props.authenticatedUser) && (
```